workbench/src/dialogs.c

```diff
@@ -443,7 +443,7 @@ gboolean dialogs_workbench_settings(WORKBENCH *workbench)
 	ui_table_add_row(GTK_TABLE(table), 1, w_enable_live_update, NULL);
 	gtk_widget_set_tooltip_text(w_enable_live_update,
 		_("If the option is activated (default), then the list of files and the sidebar"
-		  " will be updated automatically if a file or directory is created, removed or renamed."
+		  " will be updated automatically if a file or directory is created, removed or renamed. "
```

Comment:
Ideally the leading space here would also be on the previous line, but that's OK with this.

Reply:
Will do it?

Reply:
Ops, merged.

Reply:
Did see the leading space to late. Will change it directly